app/javascript/article-form/articleForm.jsx

```diff
@@ -261,26 +276,21 @@ export default class ArticleForm extends Component {
 
   onClearChanges = e => {
     e.preventDefault();
-    // eslint-disable-next-line no-restricted-globals
-    const revert = confirm(
+    // eslint-disable-next-line no-restricted-globals, no-alert
```

Comment:
`no-restricted-globals` can be removed...

Reply:
Nice catch! 👍